bon-macros/src/builder/builder_gen/member.rs

```diff
@@ -109,6 +109,9 @@ pub(crate) struct MemberParams {
 
     /// Rename the name exposed in the builder API.
     pub(crate) name: Option<syn::Ident>,
+
+    /// Whether to generate a public reference getter method for this member.
+    pub(crate) get: darling::util::Flag,
```

Comment:
I'm re-thinking that maybe a better name for this could be `#[builder(getter)]`. I think this way it's easier for people not familiar with `bon` to understand what the attribute does.

The options for the getter (if and when we add them) will be inside of parentheses like:

```rust
#[builder(
    getter(
        vis = "pub(crate)",

        name = override_name,
        
        // Return a `Copy` of the value
        copy,
        // ...
    )
)]
```

Reply:
used getset as reference example

Reply:
Yeah, in `getset` the `get` attribute is well-known because it's their primary API, but in `#[builder]` it's going to be used rarely, so a bit more verbose but descriptive name is better IMHO